sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -240,16 +240,6 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       Seq(Row("1"), Row("2")))
   }
 
-  test("SPARK-11226 Skip empty line in json file") {
```

Comment:
I removed the test because it is not relevant to the default mode `PERMISSIVE` any more. And the `SQLQuerySuite` is not perfect place for it.

Reply:
Where is it moved to then? Does that mean we don't have a regression test for SPARK-11226 anymore?